deps/acelite/ace/WFMO_Reactor.cpp

```diff
@@ -1403,8 +1403,8 @@ ACE_WFMO_Reactor::register_handler_i (ACE_HANDLE event_handle,
     {
       // Note: don't change this since some C++ compilers have
       // <auto_ptr>s that don't work properly...
-      auto_ptr<ACE_Auto_Event> tmp (new ACE_Auto_Event);
-      event = tmp;
+      std::unique_ptr<ACE_Auto_Event> tmp (new ACE_Auto_Event);
+      //event = tmp;
```

Comment:
Need help for this

Reply:
@Winfidonarleyan what's the issue? I've tried to inspect this file but I don't get any warning or error from the IDE, and I can't run it since I'm on linux

Reply:
```
      std::unique_ptr<ACE_Auto_Event> tmp (new ACE_Auto_Event);
      event = std::move(tmp);
```